apps/filecoin-site/src/app/_components/SiteLayout.tsx

```diff
@@ -49,7 +49,7 @@ export function SiteLayout({
         <StructuredDataScript structuredData={ORGANIZATION_SCHEMA_BASE} />
       </head>
       <body className="bg-white font-sans text-base/5.5">
-        <main>{children}</main>
+        <main className="overflow-x-hidden">{children}</main>
```

Comment:
Uf .. feeling queazy about adding this to the the `SiteLayout` - Must we add this so high up?